src/pykeen/contrib/lightning.py

```diff
@@ -304,7 +304,7 @@ def lit_pipeline(
 @click.command()
 @lit_module_resolver.get_option("-tl", "--training-loop")
 @dataset_resolver.get_option("--dataset", default="nations")
-@options.inverse_triples_option
+@options.use_inverse_relations_option
```

Comment:
are these renames strictly necessary for this PR?

Reply:
Maybe not.

I changed it, since we are not creating any artificial triples anymore (inverse_*triples*), but only change how relations are represented (inverse_*relations*).

Reply:
Maybe you can make an own PR for this again? Since we also used inverse relations before, this might also be an appropriate name before the functional changes made in this PR.